airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/longtail-connectors.svg

```diff
@@ -0,0 +1,667 @@
+<svg width="165" height="120" viewBox="0 0 165 120" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
It looks like this may be the same image @josephkmh used in https://github.com/airbytehq/airbyte/pull/21396

Will adjust to use his instead once it's merged!

Reply:
Actually, this is a different image!